pipelines/cpBias.yaml

```diff
@@ -0,0 +1,26 @@
+description: cp_pipe BIAS calibration construction
+tasks:
+  isr:
+    class: lsst.ip.isr.isrTask.IsrTask
+    config:
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cpBiasProc'
+      doBias: False
+      doDark: False
+      doFlat: False
+      doApplyGains: False
+      doFringe: False
+      doBrighterFatter: False
+      doVariance: True
+      doLinearize: True
+      doCrosstalk: True
```

Comment:
I think crosstalk should be switched off here.

Why is this list of configuration parameters in a different order from the others below?

Reply:
Agree on CT; switched off.  This creates slight differences between gen2 and gen3 bias results (gen2 is getting this enabled somehow).

Ordering reordered.